src/Schema/Directives/AggregateDirective.php

```diff
@@ -96,6 +96,21 @@ enum AggregateFunction {
 
     public function resolveField(FieldValue $fieldValue): FieldValue
     {
+        $modelArg = $this->directiveArgValue('model');
+        if (is_string($modelArg)) {
+            $fieldValue->setResolver(function ($root, array $args, GraphQLContext $context, ResolveInfo $resolveInfo) use ($modelArg) {
+                /** @var EloquentBuilder $query */
```

Comment:
```suggestion
```

Reply:
Wouldn't that remove autocomplete of IDE too?